package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-loop",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
see the other comment. but since 3.1 is supposed to be backwards compatible with v2, i think we only need the latest. if you have trouble upgrading to v2 though, then we can do it. it's supposed to work though.